### PR TITLE
fix #128 power-on-reset value for MSTATUS CSR

### LINKED ISSUE #128: PoR value of MSTATUS
### Bug Description

The [documented](https://github.com/openhwgroup/cve2/blob/b8c94a3e84087e230581fcdddadbea398ea19a7e/doc/03_reference/cs_registers.rst?plain=1#LL109C1-L109C1) power-on-reset value of the `mstatus` CSR (0x300) is 0x0000_1800 which expands to:

| Bit Pos  | PoR | Field     |
| ------- | ----- | -------------- |
| 21    | 1'b0  | **TW**  |
| 17    | 1'b0 | **MPRV**  |
| 12:11 | 2'b11  | **MPP**  |
| 7     | 1'b0  | **MPIE** |
| 3     | 1'b0  | **MIE**  |

However the coded PoR in the [RTL](https://github.com/openhwgroup/cve2/blob/b8c94a3e84087e230581fcdddadbea398ea19a7e/rtl/cve2_cs_registers.sv#L745) is:
```
localparam status_t MSTATUS_RST_VAL = '{mie:  1'b0,
                                          mpie: 1'b1,
                                          mpp:  PRIV_LVL_U,
                                          mprv: 1'b0,
                                          tw:   1'b0};
```
Given that [PRIV_LVL_U](https://github.com/openhwgroup/cve2/blob/b8c94a3e84087e230581fcdddadbea398ea19a7e/rtl/cve2_pkg.sv#L206) is 2'b00, this yields a PoR of 0x0000_0080.

I believe that the documented PoR value is the proper one, but the coded PoR has been in the Ibex code for three years so I am not certain.

You can show this in simulation in CORE-V-VERIF by running the `debug_test`:
```
$ make test TEST=debug_test
```
(Set the SIMULATOR and USE_ISS vars as appropriate for your installation.)

### PR DESCRIPTION
**Pull request by szbieg**

### CODE REVIEW

**Review by MikeOpenHWGroup — APPROVED** [review]
FWIW, I approve of this update as it matches the PoR state of `mstatus` in the documentation and matches what the CV32E4 cores do as well.

As the verification guy, I will defer to others to make the final determination about whether to merge this change in.

**Review by LeeHoff — APPROVED** [review]
This matches with what we discussed

**Comment by LeeHoff** [comment]
I don't think I have write access to do the merge.

**Pull request merged**
